CHANGELOG.md

```diff
@@ -1,10 +1,8 @@
 # Yii View Extension Change Log
 
-
 ## 2.0.1 under development
 
-- no changes in this release.
-
+- Enh #40: Passing content and layout parameters injections to nested view rendering (vjik)
```

Comment:
```suggestion
- Bug #40: Passing content and layout parameters injections to nested view rendering (vjik)
```

Since it was meant to work like that right from the start.

Reply:
Done